event.c

```diff
@@ -249,3 +249,44 @@ ULONGLONG GetTestEventHandle() {
 	EvtClose(result);
 	return (ULONGLONG)record;
 }
+
+ULONGLONG EnableChannel(char* ChannelName, int BufferSizeMb) {
+    //Convert char* wchar_t*.
+    int cSize = strlen(ChannelName)+1;
+    wchar_t wcChannelName[cSize];
+    for(int count = 0; count < cSize; count++) {
+        wcChannelName[count] = (wchar_t) ChannelName[count];
+    }
+
+    EVT_HANDLE hChannel = NULL;
+    EVT_VARIANT ChannelProperty;
+    DWORD dwBufferSize = sizeof(EVT_VARIANT);
+
+    hChannel = EvtOpenChannelConfig(NULL, wcChannelName, 0);
+    if (NULL == hChannel) {
+        return 1;
```

Comment:
Can we express a bit more meaning using `SetLastError` + [`GetLastError`](https://godoc.org/golang.org/x/sys/windows#GetLastError) and this [error codes](https://golang.org/src/syscall/zerrors_windows.go#L151)?